pulp_python/app/viewsets.py

```diff
@@ -100,14 +100,23 @@ def sync(self, request, pk):
         """
         Dispatches a sync task.
         """
+
+        # Get sync_mode, default to additive
+        sync_mode = request.data.get('sync_mode', 'additive').lower()
+
+        if sync_mode not in ['mirror', 'additive']:
```

Comment:
We should add a ChoiceField to the serializer, which will handle the validation for us. I already added choices as a constant. 
https://github.com/pulp/pulp/blob/3.0-dev/common/pulpcore/common/constants.py#L30-L37